README.md

```diff
@@ -18,43 +18,102 @@
 </div>
 
 ## Overview
-Collection of Theia extensions related to C/C++ development.
+
+Collection of Theia extensions, APIs, and other tools related to C/C++ development.
 
 ## Features
+
 - `@theia/cpp-debug`:
-   - Debugging support using `GDB` through the `cdt-gdb-vscode` extension.
-   - Memory view (monitor process memory during debug sessions).
-- `@theia/cpp` **deprecated** and **removed** ([past sources](https://github.com/eclipse-theia/theia-cpp-extensions/tree/184f7751f13e1ec021ccae3f076915867168d28d/packages/cpp)): 
-   - **Deprecated** C/C++ language-features extension.\
-   (Language-features should instead be contributed by VS Code extensions).
+    - Debugging support using `GDB` through the `cdt-gdb-vscode` extension.
+    - Memory view (monitor process memory during debug sessions).
+    - See the [package readme][cdbg] for further details.
+- `@theia/clangd-contexts`:
+    - API for management of [clangd][clangd] configuration files
+    - Retrieve and set contexts in one or more `.clangd` files
+    - Manage compile flags in `.clangd` files
+    - See the [package readme][cclib] for further details.
+- `@theia/clangd-contexts-cli`
+    - An example command-line tool (`clangd-context`) built on the `@theia/clangd-contexts` API for management of [clangd][clangd] configuration files in C/C++ projects.
+    - See the [package readme][cccli] for details, including a step-by-step guide to the CLI.
+- `theia-clangd-contexts-ext`
+    - An example VS Code extension demonstrating use of the `@theia/clangd-contexts` API.
+    - See the [package readme][ccvsx] for details, including a step-by-step guide to the extension UI.
+- `@theia/cpp` **deprecated** and **removed** ([past sources][cpp]):
+    - **Deprecated** C/C++ language-features extension.\
+        (Language-features should instead be contributed by VS Code extensions).
```

Comment:
I don't know why this is here or who might miss it.

This item isn't related to the feature at hand, and removing it would lose the link to past sources. Perhaps we could raise a new issue to remove this from the README if it now seems to have been long enough since the code in question was itself removed?